src/Features/CSharp/Portable/Completion/CompletionProviders/SnippetCompletionProvider.cs

```diff
@@ -159,7 +159,7 @@ SyntaxKind.EndRegionKeyword or
             var snippets = service.GetSnippetsIfAvailable();
             if (context.CompletionOptions.ShouldShowNewSnippetExperience(context.Document))
             {
-                snippets = snippets.Where(snippet => !s_builtInSnippets.Contains(snippet.Shortcut));
+                snippets = snippets.Where(snippet => s_builtInSnippets.Contains(snippet.Shortcut));
```

Comment:
bit: consider rename this to something like `s_builtInSnippetsWithoutReplacement`